references/detection/presets.py

```diff
@@ -3,7 +3,7 @@
 
 
 class DetectionPresetTrain:
-    def __init__(self, data_augmentation, hflip_prob=0.5, mean=(123.0, 117.0, 104.0)):
+    def __init__(self, *, data_augmentation, hflip_prob=0.5, mean=(123.0, 117.0, 104.0)):
```

Comment:
Keyword args are enforced in all presets but this was missed during Multi-weight support porting.